v2/api/containerservice/customizations/managed_cluster_extensions.go

```diff
@@ -17,7 +17,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/conversion"
 
-	containerservice "github.com/Azure/azure-service-operator/v2/api/containerservice/v1api20230201/storage"
+	containerservice "github.com/Azure/azure-service-operator/v2/api/containerservice/v1api20230701/storage"
```

Comment:
Don't we need customized conversion for the property that disappeared and re-appeared? Where is it?